# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocfft/hotfix-libdir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocFFT-rocm-6.3.3/library/CMakeLists.txt.vanilla	2025-04-01 21:31:20.560616249 +0200
+++ rocFFT-rocm-6.3.3/library/CMakeLists.txt	2025-04-01 21:33:00.019167719 +0200
@@ -33,6 +33,8 @@
 
 set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG ${CMAKE_CXX_FLAGS_DEBUG}")
 
+set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -L/opt/rocm/lib64 -L/opt/rocm/lib")
+
 # Print out compiler flags for viewing/debug
 if( BUILD_VERBOSE )
   message( STATUS "rocfft_VERSION: ${rocfft_VERSION}" )
--- rocFFT-rocm-6.3.3/library/src/CMakeLists.txt.vanilla	2025-04-01 21:35:41.340229580 +0200
+++ rocFFT-rocm-6.3.3/library/src/CMakeLists.txt	2025-04-01 21:36:38.878422398 +0200
@@ -302,6 +302,8 @@
   solutions.cpp
 )
 
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -L/opt/rocm/lib64 -L/opt/rocm/lib")
+
 # compilation of tuning helper object
 add_library( rocfft-tuning-helper OBJECT
   tuning_helper.cpp
